packages/react-router-dom-v5-compat/jest.config.js

```diff
@@ -9,6 +10,13 @@ module.exports = {
   setupFiles: ["./__tests__/setup.ts"],
   moduleNameMapper: {
     "^@remix-run/router$": "<rootDir>/../router/index.ts",
+    "^@remix-run/web-blob$": require.resolve("@remix-run/web-blob"),
```

Comment:
ideally you should provide CJS entry points for `node` and then we'd be able to avoid this. Right now there are both CJS and ESM entry points for `node`, but `browser` only has ESM

Reply:
@SimenB I guess https://github.com/remix-run/web-std-io/pull/43 is what you're referring to?

Reply:
yep 🙂